src/test/java/core/basesyntax/service/TransactionConverterImplTest.java

```diff
@@ -0,0 +1,107 @@
+package core.basesyntax.service;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertThrows;
+
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.util.List;
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.Test;
+
+class TransactionConverterImplTest {
+    private static final String TEST_FILE_0 = "src/test/resources/ConvTest0.csv";
+    private static final String TEST_FILE_1 = "src/test/resources/ConvTest1.csv";
+    private static final String TEST_FILE_2 = "src/test/resources/ConvTest2.csv";
+    private static final String TEST_FILE_3 = "src/test/resources/ConvTest3.csv";
+    private static final String TEST_FILE_4 = "src/test/resources/ConvTest4.csv";
+    private static final String TEST_FILE_5 = "src/test/resources/ConvTest5.csv";
+    private static final String TEST_FILE_6 = "src/test/resources/Blank.csv";

```

Comment:
you missed a chance to use a file/constant name to make this test class more readable -VALID_FILE, INVALID_HEADER_FILE etc